res/linux/mixxx.metainfo.xml

```diff
@@ -64,7 +64,7 @@
   </screenshots>
   <url type="homepage">https://mixxx.org</url>
   <url type="bugtracker">https://bugs.launchpad.net/mixxx</url>
-  <url type="donation">https://www.mixxx.org/support</url>
+  <url type="donation">https://mixxx.org/#opensource</url>
```

Comment:
We could set up a dedicated donation page on our website where we say what we do with the donations, etc.
But that's probably more work :wink:

If you use https://mixxx.org/donate here, we could swap out the redirect for such a page at some point.

Reply:
done, thanks for the redirect